Task7FinalReact/valery-youtube-searcher/src/reducers/videosReducer.js

```diff
@@ -0,0 +1,54 @@
+const SET_VIDEOS = "SET_VIDEOS"
+const ADD_VIDEOS = "ADD_VIDEOS"
+const SET_IS_FETCHING = "SET_IS_FETCHING"
+const SET_FETCH_ERROR = "SET_FETCH_ERROR"
+const SET_NEXT_PAGE_TOKEN = "SET_NEXT_PAGE_TOKEN"
+
+const defaultState = {
+    items: [],
+    isFetching: false,
+    isFetchError: false,
+    nextPageToken: ''
+}
+
+
+export default function videosReducer(state = defaultState, action) {
+    switch (action.type) {
+        case SET_VIDEOS:
+            return {
+                ...state,
+
+                items: action.payload,
+                isFetching: false
+            }
+        case ADD_VIDEOS:
+            return {
+                ...state,
+                items: [...state.items, ...action.payload],
+                isFetching: false
+            }
+        case SET_IS_FETCHING:
+            return {
+                ...state,
+                isFetching: action.payload
+            }
+        case SET_FETCH_ERROR:
+            return {
+                ...state,
+                isFetchError: action.payload
```

Comment:
You should reset `isFetchError` in other actions, otherwise it would be `true` all the time.